public/app/plugins/datasource/cloudwatch/specs/datasource.jest.ts

```diff
@@ -1,32 +1,33 @@
 import '../datasource';
-import { describe, beforeEach, it, expect, angularMocks } from 'test/lib/common';
-import helpers from 'test/specs/helpers';
+import { TemplateSrvStub } from 'test/specs/helpers';
 import CloudWatchDatasource from '../datasource';
 import 'app/features/dashboard/time_srv';
+import * as dateMath from 'app/core/utils/datemath';
 
 describe('CloudWatchDatasource', function() {
-  var ctx = new helpers.ServiceTestContext();
-  var instanceSettings = {
+  let instanceSettings = {
     jsonData: { defaultRegion: 'us-east-1', access: 'proxy' },
   };
+  let templateSrv = new TemplateSrvStub();
```

Comment:
I think you can use jest mock here like you've did for postgres, mysql and recently elasticsearch. Or is there something I'm missing?

Reply:
It seems like I forgot that! Thanks!